WaterLog_Backend/Controllers/SegmentEventsController.cs

```diff
@@ -82,22 +82,31 @@ public async Task AddSegmentEvent([FromBody] SegmentEventsEntry value)
         //Routing to get all currently opened events
         [Route("GetAlerts")]
         public async Task<List<GetAlerts>> GetAlerts()
+
         {
             try
             {
                 //Get all leaks first
-                var leaks = await _db.SegmentLeaks.Where(a => a.ResolvedStatus == EnumResolveStatus.UNRESOLVED).ToListAsync();
+                var leaks = await _db.SegmentLeaks.ToListAsync();
+                leaks = leaks.OrderByDescending(a => a.OriginalTimeStamp).OrderByDescending(a => a.ResolvedStatus).ToList();
```

Comment:
You have assumed you have gotten data from SegmentLeaks do a null check with appropriate error handling